x-pack/plugins/cloud_security_posture/public/components/compliance_score_bar.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiToolTip, useEuiTheme } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { calculatePostureScore } from '../../common/utils/helpers';
+import { statusColors } from '../common/constants';
+
+export const ComplianceScoreBar = ({
+  totalPassed,
+  totalFailed,
+}: {
+  totalPassed: number;
+  totalFailed: number;
+}) => {
+  const { euiTheme } = useEuiTheme();
+  const complianceScore = calculatePostureScore(totalPassed, totalFailed);
+
+  return (
+    <EuiFlexGroup
+      gutterSize="none"
+      alignItems="center"
+      justifyContent="flexEnd"
+      style={{ gap: euiTheme.size.s }}
```

Comment:
Hey, I see in some places of the CSP plugin, we are using `style` react prop, and in others, we use `css` from emotion. As a team, we should establish a standard for styling, and I advocate using emotion as a stronger standard because it performs better on browser rendering.

For example, in some parts of our plugin, components rendered using style are rendered in the client as HTML tags with the style attribute.

![image](https://user-images.githubusercontent.com/19270322/213313131-caeb10f6-7a4c-4cb8-8dd9-58ce258d170e.png)


And components rendered using emotion have their styles converted to classes hashed in a way that the same CSS object will generate the same class name, so the HTML tags will have class attributes pointing to these generated classes, providing a faster rendering since it will have fewer objects for the browser to parse and will reuse common styles.

![image](https://user-images.githubusercontent.com/19270322/213313703-637bf3d9-7651-4b16-981b-4d1afad0fd67.png)

![image](https://user-images.githubusercontent.com/19270322/213314226-b9b5290a-03e5-4511-9b62-7342dd749292.png)

Reply:
good thinking @opauloh! i like it 🥇

Reply:
That's a good comment @opauloh, the real reason why we have `style` props is just because when we started working we weren't sure what is the best practice in Kibana. we used `styled-components`, `emotions`, and sometimes just the `style` prop due to its ease of use.
While I don't think the performance case is a real issue here, there is certainly space for quick and easy optimization.

We should open a task to do a full swipe over our entire plugin, removing all `style` props and using emotion `css` instead. should be a great fit for a quick wins day.

Reply:
Thanks for providing the context around that, @JordanSh. I agree that this is a good candidate to be discussed and addressed in the quick wins day, so don't need to worry about it for this PR